test/examples/nodejs_postgresql_test.go

```diff
@@ -0,0 +1,539 @@
+package examples_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/redhat-developer/service-binding-operator/pkg/apis/apps/v1alpha1"
+	"github.com/redhat-developer/service-binding-operator/test/examples/util"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
+	exampleName = "nodejs_postgresql"
+	operatorsNS = "openshift-operators"
+	oprName     = "service-binding-operator"
+	appNS       = "service-binding-demo"
```

Comment:
Acronyms should have a consistent case (`As an example: ServeHTTP not ServeHttp`)... so `appNS` and not `appNs`.

Reply:
I completely agree with the consistent case when dealing with acronyms, but I then I should raise the point there might be distinction between Name Space or Namespace; some people including myself read "Namespace" where "Ns" is not an acronym but an abbreviation, whereas it seems your view point seems to read it as "Name Space", where "NS" would be an acronym.

In [Wikipedia](https://en.wikipedia.org/wiki/Namespace) it is written "Namespace", and [other sources](https://www.lexico.com/definition/namespace) refer to it as a noun, convincing me it should be treated as an abbreviation instead of an acronym.